easybuild/easyconfigs/p/prereq/prereq-tf-2.4-CrayGNU-21.09.eb

```diff
@@ -0,0 +1,65 @@
+# contributed by sarafael (CSCS)
+easyblock = 'PythonBundle'
+
+name = 'prereq'
+version = 'tf-2.4'
+
+homepage = 'https://pypi.python.org/pypi'
+description = "This module is a bundle of Python packages on Cray systems based on the module cray-python"
+
+toolchain = {'name': 'CrayGNU', 'version': '21.09'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('cray-python', EXTERNAL_MODULE),
+    ('numpy', '1.17.2'),
```

Comment:
@rsarm The module `cray-python` already provides `numpy` 1.20.1; if you want to load a module that we build due to performance reasons, there is the more recent version 1.21.4 available with the corresponding easyconfig file